driver/pace/README.md

```diff
@@ -1,17 +1,17 @@
-# pace-driver
+# pace
 
 This package provides command-line routines to run the Pace model, and utilities to write model driver scripts.
 
-We suggest reading the code in the examples directory, or taking a look at `pace/driver/run.py` to see how the main entrypoint for this package works.
+We suggest reading the code in the examples directory, or taking a look at `driver/pace/driver/run.py` to see how the main entrypoint for this package works.
```

Comment:
Did the structure change to driver/pace/run.py or is it still driver/pace/driver/run.py?